src/domain/event/hooks/useUpdateImageIfNeeded.tsx

```diff
@@ -36,21 +40,28 @@ const useUpdateImageIfNeeded = (): UpdateImageIfNeededState => {
   const { user } = useUser();
   const [updateImage] = useUpdateImageMutation();
 
-  const cleanAfterUpdate = async () => {
```

Comment:
Changes in this file could probably be in their own commit.

Reply:
These changes are now moved to a separate commit